.github/workflows/nightly.yml

```diff
@@ -20,13 +20,49 @@ jobs:
     steps:
     - uses: actions/checkout@v1
 
+    - name: Clear the environment
+      run: |
+        sudo kubectl delete pod --all || true
+        sudo kubeadm reset -f || true
+        for service in kubectl kubelet containerd epm
+        do
+          sudo systemctl stop $service || true
+          sudo systemctl disable $service || true
+        done
+        sudo pkill -9 kube-apiserver || true
+        sudo pkill -9 kube-controller-manager || true
+        sudo pkill -9 kube-proxy || true
+        sudo pkill -9 kube-scheduler || true
+        sudo yum remove -y occlum || true
+        sudo rpm -e shim-rune || true
+        sudo rpm -e rune || true
+        sudo rpm -e epm || true
+        sudo yum remove -y kube* || true
+        sudo rm -rf  $WORK_DIR
+        sudo rm -fr /etc/containerd
+        sudo rm -fr /etc/inclavare-containers/
+        sudo rm -rf ~/.kube/
+        sudo rm -rf /etc/kubernetes
+        sudo rm -fr /etc/epm
+        sudo rm -fr /var/lib/etcd
+        sudo rm -f /etc/yum.repos.d/occlum.repo
+        sudo rm -f /etc/yum.repos.d/kubernetes.repo
+        sudo ip link set cni0 down || true
+        sudo ip link delete cni0 || true
+        sudo ip link set flannel.1 down || true
+        sudo ip link delete flannel.1 || true
+        sudo ip -all netns del || true
+        sudo ps -ef | grep containerd-shim-rune-v2 | awk '{print $2}' | xargs kill -9 || true
+        sudo systemctl restart docker
+
```

Comment:
So waiting 8 pods for 30min is still necessary?

Reply:
30min is the timeout value.
As long as 8 pods are running, it will proceed to the next step.
If the environment is broken and 8 pods will never be all running, time-out 30min will prevent endless checks as showed.
```
          while true; do
            count=$(kubectl get pod -A | grep -c "1/1")
            if [ $count -eq 8 ]; then
              break
            fi
            sleep 5
          done
```

Reply:
In #436, I just make the timeout value from 10 min to 30 min. I think both are acceptable.

Reply:
I mean it was the moment when you didn't know the root cause and thus decided to extend timeout to 30min. If you ware know the root cause, is it still necessary?